packages/gasket-preset-nextjs/test/preset-prompt.test.js

```diff
@@ -19,11 +19,51 @@ describe('presetPrompt', () => {
   it('does not prompt if context.typescript exists', async () => {
     mockContext.typescript = true;
     await presetPrompt({}, mockContext, mockPrompt);
-    expect(mockPrompt.prompt).not.toHaveBeenCalled();
+    const first = mockPrompt.prompt.mock.calls[0][0][0];
+    expect(mockPrompt.prompt).toHaveBeenCalledTimes(4);
+    expect(first.name).toEqual('useAppRouter');
   });
 
   it('prompts for typescript', async () => {
     await presetPrompt({}, mockContext, mockPrompt);
+    const first = mockPrompt.prompt.mock.calls[0][0][0];
     expect(mockPrompt.prompt).toHaveBeenCalled();
+    expect(first.name).toEqual('typescript');
+  });
+
+  it('prompts for appRouter', async () => {
+    await presetPrompt({}, mockContext, mockPrompt);
+    const first = mockPrompt.prompt.mock.calls[1][0][0];
+    expect(mockPrompt.prompt).toHaveBeenCalled();
+    expect(first.name).toEqual('useAppRouter');
+  });
+
+  it('prompts for nextServerType', async () => {
+    await presetPrompt({}, mockContext, mockPrompt);
+    const first = mockPrompt.prompt.mock.calls[2][0][0];
+    expect(mockPrompt.prompt).toHaveBeenCalled();
+    expect(first.name).toEqual('nextServerType');
+  });
+
+  it('prompts for nextDevProxy when defaultServer', async () => {
+    mockContext.nextServerType = 'defaultServer';
+    await presetPrompt({}, mockContext, mockPrompt);
+    const first = mockPrompt.prompt.mock.calls[2][0][0];
+    expect(mockPrompt.prompt).toHaveBeenCalled();
+    expect(first.name).toEqual('nextDevProxy');
+  });
+
```

Comment:
Maybe just add a test case for - `Does not prompt for server if nextServerType === defaultServer` then 👍